internal/node/node_patches.js

```diff
@@ -483,23 +484,36 @@ exports.patcher = (fs = fs$1, root) => {
         }
     }
 };
-exports.escapeFunction = (root) => {
-    // ensure root is always absolute.
+exports.escapeFunction = (root, guards) => {
+    // ensure root & guards are always absolute.
     root = path.resolve(root);
+    guards = [...guards].map(g => path.resolve(g));
```

Comment:
Why clone `guards` just to invoke `.map`? 🤔

Reply:
good point :)